website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -37,9 +37,15 @@ cd $HOME/projects/memcached-operator
 # so all API groups will be <group>.example.com
 operator-sdk init --domain example.com --repo github.com/example/memcached-operator
 ```
+API groups are a mechanism to group portions of the Kubernetes API. You're probably already familiar with
```

Comment:
My ihmo about this one is: 

> API groups are a mechanism to group portions of the Kubernetes API. You're probably already familiar with some of the core Kubernetes API groups, such as `apps` or `rbac.authorization.k8s.io`. You should name your domain to group your resource types in meaningful group(s), both for ease of understanding and because these groups determine how access can be controlled to your resource types using RBAC. 

I am not sure if I can go along with this explanation. We do not create meaningful names for we are able to use RBAC. 

Note that k8s is a REST API such as any other. When we extend the API and create a CRD we are creating endpoints. Therefore, RBAC is a mechanism that defined the permissions to request GET/POST/PATCH/PUT these endpoints.

> For more information, see [the core Kubernetes docs](https://kubernetes.io/docs/reference/using-api/#api-groups) and [the Kubebuilder docs](https://book.kubebuilder.io/cronjob-tutorial/gvks.html).

So, I do not think that the above explanation addresses the question:
- Why do I need to inform a domain? How the domain is used? 
- Why I can have only one domain with the tool for now?
- What happens if I want to reconcile (create controllers) using the subcommand responsible for ( create api ) using APIs from another domain like external types ( owned by other projects ) or (core types) 

Reply:
RBAC is relevant because that's the primary way permissions to access an operator's API is going to be controlled, so the shape and contents of your domains determines what granularity of access you can have to those APIs. I assume an operator developer doesn't really care that in reality this is making REST endpoints exist on a webserver somewhere on the cluster because they don't interact with that abstraction in any way. 

I don't understand out the multi-domain stuff is relevant here at all, at least until it's written and kubebuilder and we bump the dependency here.

Reply:
I am totally ok with we change the text but I think would be great if we came back to the original motivation and ensure that we are properly adding meaningful info here. Therefore, I do not think that we should add `domain to group your resource types in meaningful group(s),` at all. That is not accurate.